updater/updater/admin.go

```diff
@@ -16,6 +18,23 @@ var ErrNotModified = errors.New("config not modified")
 
 var templateRe = regexp.MustCompile(`\$\{(.*?)\}`)
 
+var clientWithClientCerts *http.Client
```

Comment:
Unfortunately, this implementation is not compatible with onlineconf-csi-driver. Within one onlineconf-csi-driver instance, several updater instances coexist, each with its own config (and of course there may be different client certificates). I think we should use one transport for each client certificate. It's probably best to use a dictionary by public key.
Another possible solution is to remove http.Transport (or even http.Client) as an Updater option instead of ClientCertificates and leave certificate processing entirely on the side of the main package and onlineconf-csi-driver. Perhaps I like this option even more due to its simplicity.

Reply:
Can I add new type `AdminClient` and init it in NewUpdater ? 
```
type Updater struct {
	config UpdaterConfig
        client AdminClient
	mtime  string
	done   chan struct{}
}
```
AdminClient will store *http.Client  and AdminConfig